eng/common/scripts/Create-APIReview.ps1

```diff
@@ -110,11 +114,15 @@ foreach ($pkgName in $responses.Keys)
                 $FoundFailure = $True
                 if ($respCode -eq '201')
                 {
-                    Write-Error "Automatic API Review approval is pending for package $($PackageName)"
+                    Write-Host "Package version $($version) is GA and automatic API Review is not yet approved for package $($PackageName)."
+                    Write-Host "Build and release is not allowed for GA package without API review approval."
+                    Write-Host "Please reach out to API review approvers and share above link to review for approval."
+                    Write-Host "You will need to queue another build to proceed further after API review is approved"
+                    Write-Host "You can reach out to Azure SDK engineering systems on teams channel if review is in approved status"
                 }
                 else
                 {
-                    Write-Error "Failed to create API Review for package $($PackageName)"
+                    Write-Error "Failed to create API Review for package $($PackageName). Please reach out to Azure SDK engineering systems on teams channel and share this build details."
```

Comment:
I think it would be better to have an ```aka.ms``` link here to documentation so you've got the chance to intercept folks with frequently asked questions (you know its going to happen 😄).

Reply:
Thanks. I am working on updating FAQ as well as creating a new DL for arch board approvers based on feedback from archboard to be included in logs if developer needs emergency approval from arch board team.

Reply:
No worries. I'll approve this now in anticipation of an aka.ms link. I didn't have any other feedback.